evap/staff/fixtures/test_user_bulk_delete_file.txt

```diff
@@ -0,0 +1,4 @@
+testuser1
+testuser2
+contributor
+tesuserdoesnotexist
```

Comment:
`testuserdoesnotexist` - although it's not important because it does not exist ;)